src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private double area;
```

Comment:
We don't need this field because we have to obtain area dynamically with some `getArea` method